pkg/controllers/routing/network_routes_controller.go

```diff
@@ -613,7 +613,7 @@ func (nrc *NetworkRoutingController) enableForwarding() error {
 		return fmt.Errorf("Failed to run iptables command: %s", err.Error())
 	}
 	if !exists {
-		err := iptablesCmdHandler.Insert("filter", "FORWARD", 1, args...)
+		err := iptablesCmdHandler.Append("filter", "FORWARD", args...)
```

Comment:
This doesn't really impact our use case, but felt that it's worth the reminder that this was specifically set a while back for handling with RedHat variant firewall engines in #596

Reply:
Also, before the method call that was used to append was `AppendUnique`, not sure how it wouldn't be unique, but unless there is a performance cost, AppendUnique seems reasonable.

Reply:
thanks @aauren for the review. Good catch on #596 i totally forgot. 

I have taken out the part of the PR which was modiying the rules that enable forwarding. So it looks like based distro there may be still race condition. I am going to take it up seperate PR. For now will open an issue.